switmt_to_iso20022/constant_variables.bal

```diff
@@ -0,0 +1,11 @@
+const string[][] DETAILS_CHRGS = [["BEN", "CRED"], ["OUR", "DEBT"], ["SHA", "SHAR"]];
+const string PARTY_IDENTIER_ID = "id";
+const string PARTY_IDENTIFIER_CODE = "cd";
+const string DEFAULT_NUM_OF_TX = "1";
```

Comment:
`string[][]` and `string` types can be removed

Reply:
fixed in `601ea2a`